tests/output/coverage/units=controller=docker-default=python-3.8=coverage.67e1f73a4ab9.100.066554

```diff
@@ -0,0 +1 @@
+!coverage.py: This is a private format, don't read it directly!{"arcs":{}}
```

Comment:
Files in the `tests/output/` directory shouldn't be checked into git - these coverage files should be created in Jenkins.

Reply:
Done